tasks/release_test.yaml

```diff
@@ -0,0 +1,13 @@
+---
+- name: "Tito release test"
+  command: "tito release {{ releasers | join(' ') }} --test"
```

Comment:
Seems that this is just one config option different than `release.yaml`

Reply:
Yes, it is.

see comment below for design choice reasoning